src/components/logout/logout_container.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
```

Comment:
I wanted to make it easier to find the files. We already have half a dozen `channel.js` files and I didn't want to add more confusion to that by having half these files called `index.js`. I can add a separate index.js though so we ca get the nice `import Logout from 'components/logout';` without having to specify the full path.